packages/d3fc-data-join/test/bundleSpec.js

```diff
@@ -1,34 +1,41 @@
-import jsdom from 'jsdom';
+import { JSDOM, VirtualConsole } from 'jsdom';
+import fs from 'fs';
 
 describe('bundle', function() {
-    it('should correctly wire-up all the dependencies via their UMD-exposed globals (w/o d3-transition)', function(done) {
-        jsdom.env({
-            html: '<html></html>',
-            virtualConsole: jsdom.createVirtualConsole().sendTo({
-                error: done
-            }),
-            scripts: [require.resolve('d3/dist/d3.js'), require.resolve('..')],
-            done: (_, win) => {
-                const dataJoin = win.fc.dataJoin();
-                const selection = win.d3.select(win.document);
-                dataJoin(selection, []);
-                done();
-            }
+    it('should correctly wire-up all the dependencies via their UMD-exposed globals', function(done) {
+        const virtualConsole = new VirtualConsole().sendTo({
+            error: done
         });
-    });
-    it('should correctly wire-up all the dependencies via their UMD-exposed globals (w/ d3-transition)', function(done) {
```

Comment:
Couldn't for the life of me tell the difference between the `w/ d3-transition` and `w/o d3-transition` so removed